tools/kochdocs.nim

```diff
@@ -120,7 +120,9 @@ doc/packaging.rst
 doc/manual/var_t_return.rst
 """.splitWhitespace()
 
+# todo: auto-generate this list using `walkDirRec` + a blacklist
```

Comment:
No, I don't want this here.

This might be something for your own personal todo list, but it is not something which should bother everybody who looks at this file.
Also, this might trick others into thinking this is something which is officially wanted, which is not true. (Replace a list with a blacklist?)